cmd/upload.go

```diff
@@ -17,10 +18,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func loadDirectory(dirName string, scenarioFiles []string) []string {
+const fileSizeLimit = 1024 * 1024
+
+func checkSize(info os.FileInfo) error {
```

Comment:
I think we should apply this check for `mode.IsRegular()` as well, because a common usage pattern is `appland upload *.appmap.json`

Reply:
Perhaps, but I was wondering what's the intent of the check. Why prevent uploading from a symlink or a fifo, or a block device for that matter?

Reply:
Ok well, what I mean is check each file regardless of whether it’s specified individually, or encountered as a result of listing a directory. Particular logic around file modes is a different (but now somewhat entangled) question.

Reply:
Ah, I see what you mean now. The check was there but was misplaced. It's in correct place now.

Reply:
Can you take a quick look at my commit to increase the file size limit a bit, and also add the CHANGELOG and VERSION?